test/Transforms/PagerandoWrappers/wrapper-body.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: opt < %s -pagerando-wrappers -S | FileCheck %s
+
+define i32 @foo(i32 %arg) {
```

Comment:
Also might consider merging into the `global-and-internal.ll` test. Can use comments to explain each test.